handlers/BillTechLinkInsertHandler.php

```diff
@@ -64,12 +76,24 @@ public function notifyCustomerDataParse(array $hook_data = array())
 	public function messageaddCustomerDataParse(array $hook_data = array())
 	{
 		$customerid = $hook_data['data']['id'];
-		$link = self::getPaymentLink('balance', $customerid, ['utm_medium' => 'email']);
-		if (isset($hook_data['data']['contenttype']) && $hook_data['data']['contenttype'] == 'text/html') {
-			$balanceBtnCode = $this->createEmailButton($hook_data['mail_format'], $link);
-			$hook_data['body'] = preg_replace('/%billtech_balance_btn/', $balanceBtnCode, $hook_data['body']);
+		$appendClientInfoEnaled = ConfigHelper::getConfig('billtech.append_client_info', true);
```

Comment:
Lepiej jakby to się nazywało append_client_info.

Reply:
Sorki, chodziło mi o to żeby zmeinić client -> customer. Myślałem jedno a napisałem drugie. Nazwa zmiennej appendClientInfoEnabled jest spoko.